src/patternfly/components/Switch/switch.scss

```diff
@@ -100,7 +123,17 @@
     }
   }
 
-  .pf-c-switch__label {
+  .pf-c-switch__toggle-icon {
+    position: absolute;
+    top: var(--pf-c-switch__toggle-icon--Top);
+    display: block;
+    padding-left: var(--pf-c-switch__toggle-icon--PaddingLeft);
+    font-size: var(--pf-c-switch__toggle-icon--FontSize);
+    color: var(--pf-c-switch__toggle-icon--Color);
+  }
+
+  .pf-c-switch__label-on,
```

Comment:
if we had a single switch with just a modifier we could simplify the way we write the css too.